listenbrainz/db/stats.py

```diff
@@ -120,62 +116,19 @@ def get_user_stats(user_id, columns):
         return dict(row) if row else None
 
 
-def get_user_artists(user_id):
+def get_user_artists(user_id: int) -> Optional[UserArtistStatRow]:
     """Get top artists for user with given ID.
 
         Args:
             user_id (int): the row ID of the user in the DB
```

Comment:
no longer need types here if we have them in the method signature

Reply:
Done!